InnoViber/Controllers/ChatController.cs

```diff
@@ -36,18 +37,18 @@ public async Task<ChatViewModel> GetById(Guid id)
 
     // POST api/<ChatController>
     [HttpPost]
-    public void Post([FromBody] ChatViewModel chat)
+    public void Create([FromBody] ChatShortViewModel chat)
```

Comment:
why all controllers are synchronious?

Reply:
that have been changed in other pr which is not approved yet

Reply:
It is resolved in other pr which is not approved yet